Sources/StreamChat/Controllers/ChannelController/ChannelController.swift

```diff
@@ -941,6 +944,21 @@ public enum ListOrdering {
     case bottomToTop
 }
 
+// Describes the pinning expiration
+public struct Pin: Equatable {
```

Comment:
Can we rename this to `Pinning`? Also, it would be great to add documentation for public APIs

Reply:
Yes I added the docs to the functions, but forgot to add documentation directly to Pin 👍